example/problem.red

```diff
@@ -0,0 +1,38 @@
+import path
+import pointed
+import nat
+import int
+import s1
+import omega1s1
+import s2
+import s3-to-join
+
+let pΩ² (pA : ptype) : ptype = pΩ (pΩ pA)
+let pΩ³ (pA : ptype) : ptype = pΩ (pΩ² pA)
+
+let pΩ³/reflmap (pA : ptype) (B : type) (f : (pA.0) → B)
+  : pmap (pΩ³ pA) (pΩ³ (B , f (pA.1)))
+  =
+  ( λ c i j k → f (c i j k)
+  , refl
+  )
+
+let ps2 : ptype = (s2, base)
+let ps3 : ptype = (s3, base)
+let pjoin : ptype = (join, inl base)
+
+let test0-2 : pΩ³ ps3 .0 =
+  λ i j k → cube i j k
+
+let f3 : (pΩ³ ps3 .0) → (pΩ³ pjoin .0) =
```

Comment:
You should not need these parentheses around the types unless you really want to. (I do not object to adding them, but want to know whether there are still bugs in the parser.)